lib/video/screenRecording/desktop/firefoxrecorder.js

```diff
@@ -0,0 +1,161 @@
+'use strict';
```

Comment:
License block?  `firefoxWindowRecorder.js` with capitalization?  (I hate that `browsertime` doesn't name files, types, and imports exactly the same -- it makes grepping unnecessarily difficult.)